extra-admin/throttled/autobuild/build

```diff
@@ -0,0 +1,26 @@
+abinfo "Installing all scripts and configs ..."
+mkdir "$PKGDIR"/usr/lib/python"$ABPY3VER"/site-package/throttled -p
+install -Dvm755 "$SRCDIR"/throttled.py \
+    "$PKGDIR"/usr/lib/python"$ABPY3VER"/site-package/throttled/throttled.py
+install -Dvm755 "$SRCDIR"/mmio.py \
+    "$PKGDIR"/usr/lib/python"$ABPY3VER"/site-package/throttled/mmio.py
+install -Dvm644 "$SRCDIR"/etc/lenovo_fix.conf "$PKGDIR"/etc/throttled.conf
+install -Dvm644 "$SRCDIR"/systemd/lenovo_fix.service "$PKGDIR"/usr/lib/systemd/system/throttled.service
+
+abinfo "Installing LICENSE and README ..."
+install -Dvm644 "$SRCDIR"/LICENSE "$PKGDIR"/usr/share/doc/"$PKGNAME"/LICENSE
+install -Dvm644 "$SRCDIR"/README.md "$PKGDIR"/usr/share/doc/"$PKGNAME"/README.md
+
+
+abinfo "Creating symbolic link ..."
+# As the project name is changed to `throttled` from the legacy name `lenovo_fix` 
+# in commit 392f3f8, for compability reasons, the config file is named to 
+# `throttled.conf` and the legacy config file `lenovo_fix.conf` will be a 
+# symbolic link to `throttled.conf`.
+ln -s throttled.conf "$PKGDIR"/etc/lenovo_fix.conf
+ln -s throttled.py "$PKGDIR"/usr/lib/python"$ABPY3VER"/site-package/throttled/lenovo_fix.py
+
+abinfo "Copying all pre-compiled pycache ..."
+cp -a "$SRCDIR"/__pycache__ "$PKGDIR"/usr/lib/python"$ABPY3VER"/site-package/throttled/
+
+abinfo "All done"
```

Comment:
`abinfo`  at this line is unnecessary. 

Always add a blank line at the end of each file.